linter.py

```diff
@@ -16,20 +16,10 @@
 class Lintr(Linter):
     """Provides an interface to lintr R package."""
 
-    syntax = ['r', 'enhanced-r']
-    executable = 'R'
-    version_args = [
-        '--slave',
-        '--restore',
-        '--no-save',
-        '-e',
-        'packageVersion(\'lintr\')'
-    ]
-    version_re = r'(?P<version>\d+\.\d+\.\d+)'
-    version_requirement = '>= 0.1.0'
     defaults = {
         'linters': 'default_linters',
-        'cache': 'TRUE'
+        'cache': 'TRUE',
+        'selector': 'source.r'
```

Comment:
The convention is for R files to always have an uppercase R, is there a reason you changed this?

Reply:
From the linter settings documentation:

http://www.sublimelinter.com/en/stable/linter_settings.html#selector

> To find out what selector to use for given file type, use the “Tools > Developer > Show Scope Name” menu entry.

Doing this in an R file indicated that I should use source.r

Reply:
Ok!